### DISCUSSION TITLE
Deta CLI not logging in

### DISCUSSION THREAD

**Discussion by paluigi** [deta/cloud-docs#721] [Help] [answered]
Hi, I tried to login on the Deta CLI using the access token...both with the env variable and the token file.
I always get the following error (trying to list project as functionality test):
Error: failed to get projects: Unauthorized

How can I fix this? I cannot login from browser, it's a remote machine.

**Comment by sudo-subg** [comment, marked as answer]
Can you try creating new access token? And make sure it's `tokens` not `token` file in `.deta` folder in the home directory with {"deta_access_token":"your_access_token"} as the file content.

**Reply by paluigi** [reply]
Creating a new token, and awaiting a few minutes (the first trials after putting the new token gave the same error) fixed the issue.

I wonder if there is a way to force Deta CLI to refresh the token authentication?
